target/linux/ipq806x/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -44,6 +47,9 @@ case "$FIRMWARE" in
 	;;
 "ath10k/pre-cal-pci-0001:01:00.0.bin")
 	case $board in
+	askey,rt4230w-rev6)
+		caldata_extract "art" 0x5000 0x7f1f
```

Comment:
Note that the third parameter is the size. Looks to me like you should just use 0x2f20 like all the others.

Reply:
I thought you had to define the start and end. Fixed